lib/molecules/admin-bar/template.html

```diff
@@ -0,0 +1,25 @@
+<div class="toolbar" role="navigation" aria-label="Toolbar" tabindex="0" ng-if="vm.adminBarEnabled">
```

Comment:
Do we need some sass for this? It should be put in this folder too, so we don't have to repeat it in each project.